package main

// вернуть из функции ошибку, не подключая доп. пакетов
func main() {
	handle()
}

func handle() error {
	return nil
}
